src/Microsoft.ML.Data/Transforms/LabelIndicatorTransform.cs

```diff
@@ -12,18 +12,18 @@
 using System;
 using System.Text;
 
-[assembly: LoadableClass(typeof(LabelIndicatorTransform), typeof(LabelIndicatorTransform.Arguments), typeof(SignatureDataTransform),
-    LabelIndicatorTransform.UserName, LabelIndicatorTransform.LoadName, "LabelIndicator")]
-[assembly: LoadableClass(typeof(LabelIndicatorTransform), null, typeof(SignatureLoadDataTransform), LabelIndicatorTransform.UserName,
-    LabelIndicatorTransform.LoaderSignature)]
-[assembly: LoadableClass(typeof(void), typeof(LabelIndicatorTransform), null, typeof(SignatureEntryPointModule), LabelIndicatorTransform.LoadName)]
+[assembly: LoadableClass(typeof(LabelIndicatingTransformer), typeof(LabelIndicatingTransformer.Arguments), typeof(SignatureDataTransform),
+    LabelIndicatingTransformer.UserName, LabelIndicatingTransformer.LoadName, "LabelIndicator")]
+[assembly: LoadableClass(typeof(LabelIndicatingTransformer), null, typeof(SignatureLoadDataTransform), LabelIndicatingTransformer.UserName,
+    LabelIndicatingTransformer.LoaderSignature)]
+[assembly: LoadableClass(typeof(void), typeof(LabelIndicatingTransformer), null, typeof(SignatureEntryPointModule), LabelIndicatingTransformer.LoadName)]
 
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
     /// Remaps multiclass labels to binary T,F labels, primarily for use with OVA.
     /// </summary>
-    public sealed class LabelIndicatorTransform : OneToOneTransformBase
+    public sealed class LabelIndicatingTransformer : OneToOneTransformBase
```

Comment:
I am not convinced that we should use indicating as a verb. I don't know if indicating gives the right idea. An indicator is a common concept, but turning it into indicating might remove from that concept. Maybe binarizing? Or maybe keeping indicator, and adding mapping?

Reply:
This is not even a transformer btw

---
In reply to: [233189819](https://github.com/dotnet/machinelearning/pull/1606#discussion_r233189819) [](ancestors = 233189819)